packages/@ourworldindata/components/src/MarkdownTextWrap/MarkdownTextWrap.tsx

```diff
@@ -740,3 +705,319 @@ function MarkdownTextWrapLine({ line }: { line: IRToken[] }): JSX.Element {
         </span>
     )
 }
+
+export function convertMarkdownToIRTokens(
+    markdown: string,
+    fontParams?: IRFontParams
+): IRToken[] {
+    const ast = fromMarkdown(markdown)
+    return convertMarkdownRootToIRTokens(ast, fontParams)
+}
+
+function convertMarkdownRootToIRTokens(
+    node: Root,
+    fontParams?: IRFontParams
+): IRToken[] {
+    return node.children.flatMap((item) =>
+        convertMarkdownNodeToIRTokens(item, fontParams)
+    )
+}
```

Comment:
I think it's fine to inline this into `convertMarkdownToIRTokens` - but you decide :)